src/FlounderDropdown/driver.js

```diff
@@ -28,147 +16,44 @@ export default class FlounderDropdownDriver
     {
         // Nessie Control
         this.wrapper = wrapper;
+        this.control = wrapper.find( 'InputContainer' );
         // the 3rd party control
         this.innerFlounderComponent = wrapper.node.flounderInstance;
     }
 
-    /**
-     * Choosing Item in an opened Dropdown.
-     * @param {String|String[]} value - value or array of values of the selected
-     * items
-     * @return {Object} wrapper
-     */
-    chooseItemByValue( value )
-    {
-        return chooseItem( ( ...params ) =>
-            this.innerFlounderComponent.clickByValue( ...params ),
-            value,
-            'value',
-            this.wrapper
-        );
-    }
-
-    /**
-     * Choosing Item in an opened Dropdown.
-     * @param {String|String[]} text - text or array of texts of the selected
-     * items
-     * @return {Object} wrapper
-     */
-    chooseItemByText( text )
+    change( value = 'jkl' )
     {
-        return chooseItem( ( ...params ) =>
-            this.innerFlounderComponent.clickByText( ...params ),
-            text,
-            'text',
-            this.wrapper
-        );
-    }
-
-     /**
-     * Click dropdown option(s) by index
-     * @param {int|int[]} index - index or array of indexes of the selected
-     * items
-     * @return {Object} wrapper
-     */
-    chooseItemByIndex( index )
-    {
-        /* since the placeholder is always at index 0 in Flounder’s internal
-         * representation we increment the index (or indices) by 1
-         */
-        const internalIndex = Array.isArray( index ) ?
-            index.map( i => i + 1 ) : index + 1;
-
-        return chooseItem( ( ...params ) =>
-            this.innerFlounderComponent.clickByIndex( ...params ),
-            internalIndex,
-            'index',
-            this.wrapper
-        );
-    }
-
-    /**
-     * Removes all tags by clicking on them (only for flounder configured for
-     * tags)
-     * @return {Object} wrapper
-     */
-    removeAllTags()
-    {
-        if ( this.wrapper.prop( 'isDisabled' ) )
-        {
-            throw new Error( ERRORS.DROPDOWN_DISABLED );
-        }
-
-        if ( this.wrapper.prop( 'isReadOnly' ) )
-        {
-            throw new Error( ERRORS.DROPDOWN_READ_ONLY );
-        }
-
-        if ( !this.wrapper.prop( 'multipleTags' ) )
-        {
-            throw new Error( ERRORS.CANNOT_DESELECT_TAGS );
-        }
-
-        const tagWrapper = this.innerFlounderComponent.refs.multiTagWrapper;
-
-        const tags = Array.prototype.slice.call( tagWrapper.children, 0, -1 );
-
-        tags.forEach( el =>
-        {
-            el.children[ 0 ].click();
-        } );
-
+        this.innerFlounderComponent.onChange( value );
         return this;
     }
 
-    getFlounderAPI()
```

Comment:
`getFlounderAPI()` method should remain, and we should probably add an equivalent `getCodeMirrorAPI()` method